plfatools/aggregator.py

```diff
@@ -5,12 +5,19 @@
 import numpy as np
 
 class Aggregator():
+    """Automate the aggregation of GC results from PLFA and NLFA samples into a single csv file for analysis.
+    Attributes:
+        transform_raw_to_stacked (self, plfa_tools_output: pd.DataFrame): Reformats output from PLFA Tools, of a single sample, into proper tabular format
+        transform_stacked_to_tidy (self, df: pd.DataFrame): Accepts a Pandas DataFrame generated by "read_dir" or "read_file" and returns a new DataFrame in a tidy format
+        read_file (self, file_path: pathlib.Path): Reads a file from PLFA Tools and returns a pandas DataFrame with all data from all worksheets
+        read_dir (self, dir_path: pathlib.Path): Reads a directory path that contains multiple files from PLFA Tools and returns a pandas DataFrame with all data from all files and worksheets
+    """
```

Comment:
Because we're using ReST style and don't have any class attributes, I believe the docstring should just be:

```
"""Automate the aggregation of GC results from PLFA and NLFA samples into a single csv file for analysis.
"""